### PR TITLE
Add svc annotations support in kibana chart

### PR DESCRIPTION
**Pull request by JessicaGreben**
I would like to add service annotations for the Kibana's service resource.  Here is an example of my current use case:
```
annotations:
    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: https
    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:example.cert
    external-dns.alpha.kubernetes.io/hostname: kibana.example.com

```
These changes are based off of [this stable kong chart](https://github.com/kubernetes/charts/blob/master/stable/kong/templates/service-kong-admin.yaml).

Thanks! @compleatang

### CODE REVIEW

File: stable/kibana/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: kibana
-version: 0.2.3
+version: 0.2.4
```
Comment (timstoop) [inline stable/kibana/Chart.yaml]: When adding a new feature, please bump the minor version of the Chart.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by JessicaGreben** [comment]
[Tests were failing](https://circleci.com/gh/kubernetes/charts/6541?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) due to a Chart.yaml schema validation error.

I tracked down the error to a code change in [this commit](https://github.com/kubernetes/charts/commit/bd8016c53672be902cac7fa3b3cce08f59b3e210#diff-c8f9248493fcafc9a1dbeab13f06c7fa).

I updated the `circle/test/Chart.yaml` schema to reflect the Helm documentation on the Chart.yaml file:
https://github.com/kubernetes/helm/blob/master/docs/charts.md

**Review by timstoop — APPROVED** [review]

**Comment by timstoop** [comment]
That's a bummer... First PR in the project and you're already forced to look into the nitty-gritty details of the CI system to see why it's failing! Great catch, though.

**Comment by JessicaGreben** [comment]
@timstoop I'm a weirdo and actually love getting into the nitty-gritty of CI!  lol

**Comment by JessicaGreben** [comment]
odd my tests failed the linter, but locally:
```
$ helm lint kibana/
==> Linting kibana/
Lint OK

1 chart(s) linted, no failures
```
/shrug
Hopefully fixed now, with last push.

**Pull request merged**